packages/runtime-core/src/componentOptions.ts

```diff
@@ -640,7 +641,7 @@ export function applyOptions(
       for (const key in rawData) {
         checkDuplicateProperties!(OptionTypes.DATA, key)
         // expose data on ctx during dev
-        if (key[0] !== '$' && key[0] !== '_') {
+        if (!isReservedKey(key)) {
```

Comment:
I prefer to use the name `isReservedPrefix`

Reply:
make sence !